src/Containers/Microsoft.NET.Build.Containers/Tasks/ComputeDotnetBaseImageAndTag.cs

```diff
@@ -74,7 +74,10 @@ public sealed class ComputeDotnetBaseImageAndTag : Microsoft.Build.Utilities.Tas
     [Output]
     public string? ComputedContainerBaseImage { get; private set; }
 
-    private bool IsAspNetCoreProject => FrameworkReferences.Length > 0 && FrameworkReferences.Any(x => x.ItemSpec.Equals("Microsoft.AspnetCore.App", StringComparison.Ordinal));
+    private bool IsAspNetCoreProject =>
+        FrameworkReferences.Length > 0
+        && FrameworkReferences.Any(x => x.ItemSpec.Equals("Microsoft.AspNetCore.App", StringComparison.OrdinalIgnoreCase));
```

Comment:
this is the main fix - we had a lowercase `'N'` in `AspNetCore`. I made the check case-insensitive here as a double-safeguard.